MANIFEST.in

```diff
@@ -13,7 +13,7 @@ include qutebrowser/utils/testfile
 include qutebrowser/git-commit-id
 include LICENSE doc/* README.asciidoc
 include misc/qutebrowser.desktop
-include misc/qutebrowser.appdata.xml
+include misc/org.qutebrowser.qutebrowser.appdata.xml
```

Comment:
Actually, should this be named `org.qutebrowser.appdata.xml` instead (no double `qutebrowser`)? I'm not very familiar with the convention here

Reply:
Oh, never-mind, that's the id in the manifest itself.